charts/prometheus-xrstf-github-exporter/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
Do we really need service account then? I think we can avoid using it as far as we don't set any k8s specific perms for pods

Reply:
Removed